src/python/pants/backend/go/target_types.py

```diff
@@ -21,11 +21,38 @@
     StringField,
     Target,
     TargetGenerator,
+    TriBoolField,
     ValidNumbers,
     generate_multiple_sources_field_help_message,
 )
 from pants.util.strutil import softwrap
 
+# -----------------------------------------------------------------------------------------------
+# Build option fieklds
+# -----------------------------------------------------------------------------------------------
+
+
+class GoCgoEnabledField(TriBoolField):
+    """Enables whether Cgo support will be enabled."""
```

Comment:
"Enable Cgo support." ?

Can't use "enable" twice in the sentence...

Reply:
fixed